design-doc-template.adoc

```diff
@@ -174,13 +211,19 @@ of what the test is doing.
 
 == Community Documentation
 
-__<Describe how this feature will be documented or illustrated. Generally a feature should have documentation as part of
+////
+Describe how this feature will be documented or illustrated. Generally a feature should have documentation as part of
 the PR to wildfly main, or as a follow-up PR if the feature is in wildfly-core. In some cases, though, the feature will
-bring additional content (such as quickstarts, guides, etc.) Indicate which of these will happen.>__
+bring additional content (such as quickstarts, guides, etc.) Indicate which of these will happen.
+////
 
 == Release Note Content
 
-__<Draft verbiage for up to a few sentences on the feature for inclusion in the Release Note blog article for the release
-that first includes this feature.__
-__Example article: https://www.wildfly.org/news/2024/01/25/WildFly31-Released/.__
-__This content will be edited, so there is no need to make it perfect or discuss what release it appears in.>__
+////
+Draft verbiage for up to a few sentences on the feature for inclusion in the Release Note blog article for the release
+that first includes this feature.
+
+Example article: https://www.wildfly.org/news/2024/01/25/WildFly31-Released/
+
+This content will be edited, so there is no need to make it perfect or discuss what release it appears in.
+////
```

Comment:
Good adoc comment. Reviewers need reminders of what this area is meant to cover.